Tests iOS/MyList/SearchTests.swift

```diff
@@ -363,6 +365,19 @@ class SearchTests: XCTestCase {
         searchField.typeText("item\n")
 
         XCTAssertTrue(app.saves.searchView.hasBanner(with: "Limited search results"))
+        // TODO: Fix tests after addressing issue with other server error banner
```

Comment:
unable to write accurate tests due to this issue
https://pocket.slack.com/archives/C01UH57EJKD/p1685029047272619